server/multitenancy/tenant_resolver.ts

```diff
@@ -18,9 +18,11 @@ import { KibanaRequest } from '../../../../src/core/server';
 import { SecuritySessionCookie } from '../session/security_cookie';
 import { SecurityPluginConfigType } from '..';
 
-export const PRIVATE_TENANT: string = '__user__';
-export const GLOBAL_TENANT: string = '';
+const PRIVATE_TENANT_SYMBOL: string = '__user__';
+const GLOBAL_TENANT_SYMBOL: string = '';
 
+export const PRIVATE_TENANTS: string[] = [PRIVATE_TENANT_SYMBOL, 'private'];
+export const GLOBAL_TENANTS: string[] = ['', GLOBAL_TENANT_SYMBOL];
```

Comment:
These two elements are same. Wondering why we need both.

Reply:
actually, they are different :P

Reply:
I am confused. 

 const GLOBAL_TENANT_SYMBOL: string = '';

which is same as the first element '' in the array GLOBAL_TENANTS